client/lib/make-json-schema-parser/index.js

```diff
@@ -38,7 +37,13 @@ export class TransformerError extends Error {
  *
  * @return {Parser}                       Function to validate and transform data
  */
-export function makeJsonSchemaParser( schema, transformer = identity, schemaOptions = {} ) {
+export function makeJsonSchemaParser(
```

Comment:
These changes are oriented at reducing bundle size. With #28559, these changes will be irrelevant.

Extracted to #28587